### PR TITLE
Adds initial guidelines for security reviewers

### PR DESCRIPTION
**Pull request by gunsch**
First pass at a list of "here are questions you should ask in privacy/security/infrastructure review".

PTAL, let me know if I'm missing anything obvious.

### CODE REVIEW

File: platform/security/hosting-a-security-review.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+## Questions
+
+This list of questions is _not_ intended to be comprehensive. It _is_ intended to be comprehensively covered in a review.
+
+### Privacy
+
+#### Logs
+
+* What new log statements have the developers added? (FE or BE). This includes data sent to Sentry, Cloudwatch, Google Analytics, ...?
+* What kinds of information are being logging? Is there anything that uniquely identifies a user (PII)?
```
Comment (rianfowler) [inline platform/security/hosting-a-security-review.md]:
```suggestion
* What kinds of information are logged in the new logging? Is there anything that uniquely identifies a user (Personally Identifiable Information)?
```

File: platform/security/hosting-a-security-review.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+## Questions
+
+This list of questions is _not_ intended to be comprehensive. It _is_ intended to be comprehensively covered in a review.
+
+### Privacy
+
+#### Logs
+
+* What new log statements have the developers added? (FE or BE). This includes data sent to Sentry, Cloudwatch, Google Analytics, ...?
```
Comment (rianfowler) [inline platform/security/hosting-a-security-review.md]:
```suggestion
* Are there any new logging statements? (FE or BE). This includes data sent to Sentry, Cloudwatch, Google Analytics, ...?
```

File: platform/security/hosting-a-security-review.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+#### User tracking / identity
+
+* Are there any new cookies?
+* Are there any new uses of existing cookies?
+
+### Security
+
+* How are systems authenticating with one another?
+* Is anyone bypassing any obvious built-in protections? (CSRF protection in form submissions, auto-escaping in FE template rendering systems, use of `innerHTML`)
```
Comment (rianfowler) [inline platform/security/hosting-a-security-review.md]: We need to train FE on these

**Review by rianfowler — APPROVED** [review]

**Comment by gunsch** [comment]
@johnpaulashenfelter @wyattwalter any thoughts? I'll plan to merge this tomorrow if I don't hear any other feedback.

**Comment by gunsch** [comment]
Added some questions about rollout plan (from this morning's security review). Merging!

**Pull request merged**